colibri/src/icons.rs

```diff
@@ -405,16 +405,19 @@ pub async fn query_icon_remotely(
                 debug!("Detected potential Uniswap V3/V4 position NFT: {}", asset_id);
                 if let Some(blockchain) = SupportedBlockchain::from_chain_id(asset_info.chain_id) {
                     let inquirer = evm_inquirer_manager.get_or_init_inquirer(blockchain).await;
-                    if let Some((icon_bytes, extension)) = query_uniswap_position_icon(
-                        asset_info.chain_id,
-                        token_id,
-                        asset_info.contract_address,
-                        inquirer,
-                    )
+                    // Uniswap positions are only on EVM chains
+                    if let AssetAddress::Evm(evm_address) = asset_info.contract_address {
+                        if let Some((icon_bytes, extension)) = query_uniswap_position_icon(
+                            asset_info.chain_id,
+                            token_id,
+                            evm_address,
+                            inquirer,
+                        )
```

Comment:
seems that this is missing an indentation

Reply:
ran `cargo fmt`.